package.json

```diff
@@ -89,7 +91,7 @@
     "proxyquire": "~1.8.0",
     "registry-mock": "^2.0.0",
     "request": "~2.81.0",
-    "sinon": "~1.17.7",
+    "sinon": "^6.1.4",
```

Comment:
We should probably update more of these. At least the dev deps as some general maintenance.

Reply:
This is an open source project, can we just get Greenkeeper to look at this repo?